lib/pechkin/connector/base.rb

```diff
@@ -6,7 +6,7 @@ class Base
         'Content-Type' => 'application/json; charset=UTF-8'
       }.freeze
 
-      def send_message(chat, message, message_desc); end
+      def send_message(chat, email, message, message_desc); end
```

Comment:
This method is abstract and shouldn't have `email` as a parameter as it leaks implementation details. We should either make `chat` parameter an object that represents different "handles" of chats. Or keep it as string and resolve inside each particular connector. I'm still leaning towards `string` option for now.